src/v0/destinations/redis/transform.js

```diff
@@ -21,8 +21,33 @@ const processValues = (obj) => {
   });
 };
 
+const isSubEventTypeProfiles = (message) => {
+  // check if profiles_model, profiles_entity, profiles_id_type are present in message.context.sources
+  const { context } = message;
+  if (!context?.sources) {
+    return false;
+  }
+  const { sources } = context;
+  return sources?.profiles_entity && sources?.profiles_id_type && sources?.profiles_model;
```

Comment:
sources?. "?" not required right, it is already checked

Reply:
Addressed, thanks!